client/engine/engine.go

```diff
@@ -546,6 +552,16 @@ func (e *Engine) getOrCreateObjective(p protocols.ObjectivePayload) (protocols.O
 
 		newObj, err := e.constructObjectiveFromMessage(id, p)
 
+		if dfo, isDF := newObj.(*directfund.Objective); isDF {
+			e.chain.Monitor(
+				dfo.C.ChannelId(),
+				dfo.C.PostFundState().Outcome.TotalAllocatedFor(types.AddressToDestination(*e.store.GetAddress())),
+				dfo.C.Total(),
+			)
+		}
+		if ddfo, isDDF := newObj.(*directdefund.Objective); isDDF {
```

Comment:
Do you think we should just remove this for now?

Reply:
Oops, yup!

Reply:
Let's merge this PR in after this is addressed.